### PR TITLE
Added endpoint for showing pets by type

### PR DESCRIPTION
**Pull request by JohnABobst**
Also updated show page for pets by type

### CODE REVIEW

File: node-group-project/server/server.js [hunk 1]
```diff
@@ -83,6 +83,13 @@ app.get('/api/v1/pettypes', (req, res) => {
   helpers.getAllFromTable("pet_types", res, pool)
 })
 
+app.get('/api/v1/pets/:pet_type', (req, res) => {
+  let queryString = "SELECT * FROM adoptable_pets LEFT JOIN pet_types ON adoptable_pets.type_id = pet_types.id WHERE type = $1";
```
Comment (prophetsfall) [inline node-group-project/server/server.js:87]: Can you explain why you've chosen a left join here? 
I'm having trouble finding a scenario where you'd want pet types that don't have an adoptable pet.

File: node-group-project/src/components/PetTypeContainer.js [hunk 2]
```diff
@@ -3,28 +3,34 @@ import { Link } from "react-router-dom";
 
 const PetTypeContainer = (props) => {
   const [adoptablePets, setAdoptablePets] = useState([]);
-
+  console.log("Pet type container")
```
Comment (prophetsfall) [inline node-group-project/src/components/PetTypeContainer.js:6]: please remove console.logs prior to submitting a PR. 

This applies to all instances in the file.

**Review by prophetsfall — CHANGES_REQUESTED** [review]
One question, one change.

File: node-group-project/server/server.js (hunk 1 above)
Comment (JohnABobst) [reply node-group-project/server/server.js:87]: That actually makes a lot of sense and I can't remember why I did a left there.  I'll change it.

**Pull request merged**